CanvasCore/CanvasCore/Helm/HelmViewController.swift

```diff
@@ -199,15 +199,17 @@ public final class HelmViewController: UIViewController, HelmScreen, PageViewEve
     
     // MARK: - Styles
     @objc public func handleStyles() {
-        switch screenConfig[PropKeys.navBarStyle] as? String {
-        case "context":
-            if let color = screenConfig.navBarColor {
-                navigationController?.navigationBar.useContextColor(color)
+        if !screenConfig.config.isEmpty {
```

Comment:
The controller often renders before any props have made it in, and setting the nav bar in that case is always wrong.